### DISCUSSION TITLE
the output of the temporal action detection？

### DISCUSSION THREAD

**Discussion by vivva** [open-mmlab/mmaction2#745] [Q&A]
I am new to here，I want to ask，can the output of the temporal action detection model be the video of the detected action category and boundary?

**Comment by ZJU-lishuang** [comment]
You can refer [this](https://github.com/open-mmlab/mmaction2/tree/master/demo#spatiotemporal-action-detection-video-demo)
In demo folder ,there already was a SpatioTemporal Action Detection Video Demo
